.github/workflows/test_precommit.yml

```diff
@@ -148,13 +148,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-20.04, ubuntu-22.04, macos-13]
+        os: [ubuntu-20.04, ubuntu-22.04, ubuntu-24.04]
         python-version: [3.9, '3.10', '3.11']
         exclude:
-          - os: macos-13
-            python-version: '3.10'
-          - os: macos-13
-            python-version: '3.11'
+          - os: ubuntu-24.04
+            python-version: '3.9'
```

Comment:
It's just a question. Why is it excluded?

Reply:
It's unrealistic somebody would use 3.9 with the latest Ubuntu version